components/TItemEdit.vue

```diff
@@ -33,7 +33,7 @@ import { computed } from '@nuxtjs/composition-api'
 import { track } from '~/plugins/firebase'
 import { useAuth } from '~/use/auth'
 import { useDoc } from '~/use/doc'
-import { useRouter } from '~/use/router'
+import { useRoute } from '@nuxtjs/composition-api'
```

Comment:
Seeing the building console error, it looks like `useRoute not found in '@nuxtjs/composition-api'`.
Are you sure this is correct?